### PR TITLE
Removed remaining hardcoded strings,dimens,colors

### PR DESCRIPTION
**Pull request by AaskaShah**
1) Removed remaining hard-coded strings,dimens,colors
2) Removed color defined in dimens

### CODE REVIEW

**Comment by sandarumk** [comment]
When creating a PR first create an issue. Create the PR if the issue is accepted only. This PR conflicts with the issue https://github.com/systers/PCSA-Android/issues/97.

File: app/src/main/res/layout/fragment_circle_of_trust.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
                 android:textColor="@color/fragment_of_trust_text_color"
                 android:src="@mipmap/ic_comrade"
                 android:textSize="@dimen/text_normal"
-                android:text="CM 2"
+                android:text="@string/fragment_circle_of_trust_com2"
```
Comment (sandarumk) [inline app/src/main/res/layout/fragment_circle_of_trust.xml]: image view should not have these tags, issue #97

**Comment by sandarumk** [comment]
You can create the issue and implement it after #97 is merged. (You do not have to wait until it is merged to create the issue) Please go through the ReadMe first. If the reported issue is a good one we will give a proceed signal.

**Comment by AaskaShah** [comment]
I apologize for this. But I sent a PR as an issue was already reported on this, #40 . I would surely make an issue before submitting PR now onwards.

**Comment by sandarumk** [comment]
#40 is a closed issue and it was assigned to yatna. Make sure before you submit  a PR, your issue is accepted and assigned to you.

**Comment by AaskaShah** [comment]
I would surely keep this in mind :)

**Comment by sandarumk** [comment]
#97 is merged. See whether this still exists in the new code. If so you submit an issue.

**Comment by AaskaShah** [comment]
Fixes #105

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -107,8 +107,15 @@
     <string name="str_cnt_mnu_share">Share</string>
     <string name="str_cnt_mnu_delete">Delete</string>
     <string name="title_activity_trustees">Trustees</string>
+    <string name="activity_trustees_text">Add the Comrade Numbers here</string>
+    <string name="hint_comrade1">Comrade 1</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: You don't have to define a new string for this comrade 1 is already there on the list.<string name="comrade1">Comrade 1</string>

**Comment by AaskaShah** [comment]
Removed them :)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -107,8 +107,9 @@
     <string name="str_cnt_mnu_share">Share</string>
     <string name="str_cnt_mnu_delete">Delete</string>
     <string name="title_activity_trustees">Trustees</string>
+    <string name="activity_trustees_text">Add the Comrade Numbers here</string>
```
Comment (sandarumk) [inline app/src/main/res/values/strings.xml]: This is also in the string.xml earlier

**Comment by AaskaShah** [comment]
I removed `<string name="comrade_title">Add the Comrade Numbers here</string>` . Also it wasn't used anywhere.

**Comment by sandarumk** [comment]
Isn't it used in the Add comrade numbers screen?

**Comment by AaskaShah** [comment]
Actually it was hardcoded , so now I added `<string name="activity_trustees_text">Add the Comrade Numbers here</string>`

**Comment by sandarumk** [comment]
Yes I found the place. You are correct. Instead of activity trustees text which is a little bit of ambiguous name use a name like add comrades text.

**Comment by AaskaShah** [comment]
Changed the name to add_comrades_text.

**Comment by sandarumk** [comment]
Great!

**Pull request merged**